test/unit/test_hub_config.py

```diff
@@ -72,7 +73,25 @@ def tasks_config() -> dict:
                     },
                 ],
                 "submissions_due": {"relative_to": "reference_date", "start": -6, "end": -3},
-            }
+            },
+            {
+                "round_name": "Round 2",
```

Comment:
It would seem that we would want to provide and look up based on the required `round_id` rather than the optional `round_name` here? https://github.com/hubverse-org/schemas/blob/0f35d3bdcb7c091183c86d552d6dd2270fdb89ea/v3.0.0/tasks-schema.json#L1436

Reply:
I'm glad you asked, because this was a point of confusion for me. In many examples I've looked at, the `round_id` is a function of some other variable (e.g., reference_date). So I wasn't sure how we'd be able to filter on the actual `round_id`  when parsing `tasks.json`, and landed on a starting point of "put in a round_name if you want to do this."

That said, the R version of this function does filter on `round_id`, so I see the argument of matching that. Unless I'm missing something (entirely possible), I don't think it would work for hubs that use `round_id_from_variable`, and maybe that's fine!

Reply:
Here's a very rough idea for hubs that potentially use `round_id_from_variable`, supposing that `tasks.json` has been read into an object called `tasks`:

```
round_ids = [ _get_round_ids(r) for r in tasks.rounds ]

def _get_round_ids(r):
    if not r.round_id_from_variable:
        return r.round_id
    
    return the union of [ task_set.task_ids[r.round_id] for task_set in r.model_tasks ]
```

Not 100% I got the details right there, but basically looking up all the possible `round_id` values when necessary.